resources/fixtures/data/results-threshold-color-codes-uncertainty.json

```diff
@@ -0,0 +1,66 @@
+{
+  "title": "FIXTURE: results - threshold - color codes - uncertainty",
+  "tool": "election_votes",
+  "department": "Visuals",
+  "publication": "nzz",
+  "acronym": "awi.",
+  "parties": [
+    {
+      "color": {
+        "colorCode": "#7b0d05"
+      },
+      "errorMargin": {
+        "bestGuess": 13
+      },
+      "name": "Partei A",
+      "percentage": 16,
+      "previous": 15
+    },
+    {
+      "color": {
+        "colorCode": "#1b53be"
+      },
+      "errorMargin": {
+        "lower": 3,
+        "bestGuess": 5,
+        "upper": 7
+      },
+      "name": "Partei B"
+    },
+    {
+      "color": {
+        "colorCode": "#009ec3"
+      },
+      "errorMargin": {
+        "lower": 80,
+        "bestGuess": 82,
+        "upper": 87
+      },
+      "name": "Tollak Party "
+    },
+    {
+      "color": {
+        "colorCode": "#a00cb3"
+      },
+      "errorMargin": {
+        "bestGuess": 12
+      },
+      "name": "Other Party"
+    }
+  ],
+  "sources": [],
+  "options": {
+    "hideUpdatedDate": false
+  },
+  "subtitle": "Aktuelle Hochrechnung von 13:30",
+  "withErrorMargin": true,
+  "errorMarginLabels": {
+    "bestGuessLabel": "Best guess",
+    "errorMarginLabel": "Fehlermarge"
+  },
+  "threshold": 5,
+  "createdDate": "2019-03-12T07:30:52.700Z",
+  "createdBy": "anna.wiederkehr@nzz.ch",
+  "updatedDate": "2019-03-12T07:33:13.235Z",
+  "updatedBy": "anna.wiederkehr@nzz.ch"
```

Comment:
Could you remove unnecessary properties createdDate, createdBy, updatedDate, updatedBy, publication, acronym and department?